src/artifacts-helper/codespaces_artifacts_helper_keyring/noxfile.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import nox
+
+os.environ.update({"PDM_IGNORE_SAVED_PYTHON": "1"})
+
+PYTHON_VERSIONS = ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
I haven't tested all the versions yet. I was planning to defer this task to CI using `nox` to generate a matrix. See https://nox.thea.codes/en/stable/cookbook.html#generating-a-matrix-with-github-actions